x/ibc/20-transfer/types/msgs.go

```diff
@@ -5,16 +5,17 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
+
+	channel "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
+	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
 	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
 )
 
-type MsgTransfer struct {
-	SourcePort    string         `json:"source_port" yaml:"source_port"`       // the port on which the packet will be sent
-	SourceChannel string         `json:"source_channel" yaml:"source_channel"` // the channel by which the packet will be sent
-	Amount        sdk.Coins      `json:"amount" yaml:"amount"`                 // the tokens to be transferred
-	Sender        sdk.AccAddress `json:"sender" yaml:"sender"`                 // the sender address
-	Receiver      sdk.AccAddress `json:"receiver" yaml:"receiver"`             // the recipient address on the destination chain
-	Source        bool           `json:"source" yaml:"source"`                 // indicates if the sending chain is the source chain of the tokens to be transferred
+type MsgRecvPacket struct {
```

Comment:
the order of the Msgs in the file is off

Reply:
Aye - fixed.